onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -1857,30 +1857,30 @@ namespace OperatorHelper
         DowncastDimensions(gsl::span(shapeData), /*out*/ m_blockShape);
 
         const uint32_t dimCount = gsl::narrow_cast<uint32_t>(m_blockShape.size());
-        m_dilations = {dimCount, 1};
-        m_pads = {dimCount * 2, 0};
-        m_strides = {dimCount, 1};
+        m_dilations.resize(dimCount);
+        std::fill(m_dilations.begin(), m_dilations.end(), 1);
+        m_pads.resize(dimCount, 0);
```

Comment:
I was `dimCount * 2` before. Should it be `m_pads.resize(dimCount * 2, 0);`?

Reply:
Its correct now!